browser/src/slideshow/LayerDrawing.ts

```diff
@@ -682,6 +683,27 @@ class LayerDrawing {
 		}
 	}
 
+	private drawTextField(slideHash: string) {
```

Comment:
Why do we need this? How did it work before without this piece?

I see it is copy pasted from https://github.com/CollaboraOnline/online/pull/12068/files#diff-eae2440a748123c5dedb24481c560854c706651d4950041914cfbf5501a4d830L620

If it's really needed would be good to share the code. Or maybe move completely to be drawn completely once?